### PR TITLE
Added jekyll configs and fixed some formatting.

### PR DESCRIPTION
**Pull request by antaresc**
- Added Gemfiles to enable local builds
- Added README
- Added gitignore
- Fixed formatting issues with Python args

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
-remote_theme: pmarsceill/just-the-docs
+theme: "just-the-docs"
+# remote_theme: pmarsceill/just-the-docs
```
Comment (shifucun) [inline _config.yml]: remove this?

File: _config.yml [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
 
 # logo: "/assets/dc.png"
```
Comment (shifucun) [inline _config.yml]: can remove this as well

File: api/python/api_key.md [hunk 3]
```diff
@@ -22,4 +22,4 @@ for more details.
 
 **Arguments**
 
-*   `api_key (str)` - The API key.
+*   `api_key` (`str`): The API key.
```
Comment (beets) [inline api/python/api_key.md]: For these -- I think it's cleaner not to escape the 'code' style multiple times (see tjann's comment on the group thread).

File: api/python/query.md [hunk 4]
```diff
@@ -50,8 +50,6 @@ Be sure to initialize the library, and specify the API key. Check the [Python li
 
 ## Examples
 
-**Examples**
-
```
Comment (beets) [inline api/python/query.md]: thanks for fixing this

**Pull request closed without merging**